toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -42,3 +60,39 @@ func copyAdditionalFiles(baseConfigPath string, additionalFiles map[string]image
 
 	return nil
 }
+
+func runScripts(baseConfigPath string, scripts []imagecustomizerapi.Script, imageChroot *safechroot.Chroot) error {
+	configDirMountPath := filepath.Join(imageChroot.RootDir(), configDirMountPathInChroot)
+
+	// Bind mount the config directory so that the scripts can access any required resources.
+	mount, err := safemount.NewMount(baseConfigPath, configDirMountPath, "", unix.MS_BIND|unix.MS_RDONLY, "", true)
+	if err != nil {
+		return err
+	}
+	defer mount.Close()
+
+	for _, script := range scripts {
+		scriptPathInChroot := filepath.Join(configDirMountPathInChroot, script.Path)
+		command := fmt.Sprintf("%s %s", scriptPathInChroot, script.Args)
+
+		// Run the script.
+		err = imageChroot.UnsafeRun(func() error {
```

Comment:
Do we need `UnsafeRun()` here? I think the intent of Unsafe is to allow two chroot operations to happen at the same time, might not be needed here since I don't think there is a nested chroot here (at least I didn't notice one lookup up the call stack).

Reply:
The imager/installutils code also uses `UnsafeRun` for running these scripts. So, that logic is mirrored here.

I think this might be because the imager tool runs most of the image build logic under the setup chroot. So, the commands run against the install chroot are "nested" chroot commands. Hence using `Run` won't work.

Reply:
That would make sense.